src/database/food.go

```diff
@@ -0,0 +1,34 @@
+package database
+
+import (
+	"foodloop/src/models"
+)
+
+func GetFoods(foodlistID string) ([]models.Food, error) {
```

Comment:
with the new schema in the other [PR](https://github.com/mahe993/foodloop-server/pull/13), theres additional column `category` for food table